src/applications/check-in/travel-claim/pages/travel-agreement/index.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import PropTypes from 'prop-types';
+
+import Wrapper from '../../../components/layout/Wrapper';
+import BackButton from '../../../components/BackButton';
+import { useFormRouting } from '../../../hooks/useFormRouting';
+
+const TravelAgreement = props => {
+  const { router } = props;
+  const { t } = useTranslation();
+
+  const { goToPreviousPage } = useFormRouting(router);
+  const finishReview = () => {
```

Comment:
I thought about removing this since it's not doing anything else but this will be where tracking will go. You could add that now or add a TODO for the tracking ticket. I think there is a separate GA ticket.